08LessonEight/homework/homework.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <title>Exercise 06</title>
+    <link type="text/css" rel="stylesheet" href="css/homework-style.css" />
+</head>
+<body>
+    <main>
+        <section class="mainframe clearfix">
+            <div class="infobox clearfix">
```

Comment:
why would you apply `clearfix` to the element that is floating? - Same with the one on line 19 and 27.

`Clear-fix` is applied to the parent when all child elements are floating, no to the elements floating themselves.